src/java/org/apache/cassandra/io/sstable/format/SSTableFormat.java

```diff
@@ -54,9 +50,14 @@
  */
 public interface SSTableFormat<R extends SSTableReader, W extends SSTableWriter>
 {
-    int ordinal();
     String name();
 
+    @Deprecated
+    default Type getType()
```

Comment:
only used for tests, and is redundant as `name` and `this.getClass()` is known by holding reference to this